### ISSUE TITLE
magenta/models/image_stylization result problem

### ISSUE THREAD

**Issue by lisiping0817** [tensorflow/magenta#1285]
![image](https://user-images.githubusercontent.com/41105367/45192916-e9533280-b27d-11e8-9fa8-01c8b5b4f167.png)
   
i train the model again and result is not good  in the transfer picture ,and train step i follow the author ,so i want to know how make the result better.
thank you

**Comment by vdumoulin** [comment]
Hi @lisiping0817, thanks for reaching out! Can you provide me with more information on the problem you're having?

* What command did you run to create the style dataset?
* What style images did you provide for training? What are their resolution?
* What command did you run to train the model? In particular, did you use the default content/style loss hyperparameters?
* What command did you run to perform the stylization?
* What is the resolution of the content image you stylized?
* What is it about the stylization which does not meet your expectations? Is it the grainy visual artifact, the fact that the sky is not stylized uniformly, or something else?
